<?xml version="1.0"?>
<launch>
    <param name="/gains/lookahead"  value="2.0"/>
    <param name="/gains/look_forward"  value="0.3"/>

    <param name="/look_ahead_constant"  value="2.5"/>
    <param name="/base_length"  value="2.5"/>
    <param name="/map_given"  value="False"/>

    <param name="/time_step"  value="0.1"/>
    <param name="/speed/max"  value="10.0"/>
    <param name="/speed/min"  value="2.0"/>
    <param name="/speed/target"  value="2.0"/>
    <param name="/acceleration/max"  value="15.0"/>
    <param name="/acceleration/min"  value="-15.0"/>

    <param name="/gains/proportional"  value="0.2"/>
    <param name="/gains/differential"  value="0.3"/>
    <param name="/gains/integral"  value="0.1"/>

    <param name="/rate"  value="10"/>
    <param name="/plotting"  value="1"/>
    <param name="/plotName"  value="plotname"/>


</launch>
